app/views/layouts/_navigation_links.haml

```diff
@@ -1,8 +1,8 @@
-%li{:class => is_active_link?(root_path)}= link_to "Home", root_path
 %li{:class => is_active_link?(page_path('about'))}= link_to "About", page_path('about')
-- if user_signed_in?
-  %li{:class => is_active_link?(edit_user_registration_path)}= link_to 'Edit account', edit_user_registration_path
-  %li= link_to 'Sign out', destroy_user_session_path, :method=>'delete'
-- else
-  %li{:class => is_active_link?(new_user_session_path)}= link_to 'Log in', new_user_session_path
-  %li{:class => is_active_link?(new_user_registration_path)}= link_to 'Sign up', new_user_registration_path
+%li{:class => is_active_link?(page_path('contact'))}= link_to "Contact", page_path('contact')
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace<br>Line is too long. [93/80]